### PR TITLE
FileService_DbContext

### PR DESCRIPTION
**Pull request by Inzilbeth**
FileService project is added to the solution.
DbContext is defined and the database is created locally using PM console.

### CODE REVIEW

File: DigitalOffice/Services/FileService/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "FileService": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:5001;http://localhost:5000",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
```
Comment (mekk1t) [inline DigitalOffice/Services/FileService/Properties/launchSettings.json:24]: In "applicationUrl" string ports should be changed to "98xx" where "xx" is a number from the scheme of the numbered services. Look it up in our Slack channel.

File: DigitalOffice/Services/FileService/Startup.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+        {
+            if (env.IsDevelopment())
+            {
+                app.UseDeveloperExceptionPage();
+            }
+
+            app.UseRouting();
```
Comment (mekk1t) [inline DigitalOffice/Services/FileService/Startup.cs]: There should be a string (coming as first)
app.UseHttpsRedirection()
(look it up in other services)

**Comment by lalagvana** [comment]
Please, add a description for this Pull Request - tell us about your final result there :)
You can see, how it's done in other Pull Requests.

**Review by mekk1t — COMMENTED** [review]
Need to make a few adjustments and everything's ok in my opinion.

**Review by mekk1t — APPROVED** [review]
Everything seems to be in order

**Review by lalagvana — APPROVED** [review]

File: DigitalOffice/Services/FileService/Startup.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
+using FileService.Database;
```
Comment (nmamro) [inline DigitalOffice/Services/FileService/Startup.cs:1]: Is migration automated? mb i missed sth

Comment (MihailKulikov) [reply DigitalOffice/Services/FileService/Startup.cs:1]: No

**Review by ifelseelif — APPROVED** [review]

**Review by denis-kriv — COMMENTED** [review]
Resolve conflicts please

**Review by djino97 — APPROVED** [review]

**Review by denis-kriv — APPROVED** [review]

File: DigitalOffice/Services/FileService/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            app.UseHttpsRedirection();
+            app.UseRouting();
+
+            app.UseEndpoints(endpoints =>
+            {
+                endpoints.MapControllers();
+            });
+        }
+
+        private static void UpdateDatabase(IApplicationBuilder app)
```
Comment (DarkFelas) [inline DigitalOffice/Services/FileService/Startup.cs]: why static?

**Review by DarkFelas — CHANGES_REQUESTED** [review]

File: DigitalOffice/Services/FileService/Startup.cs (hunk 4 above)
Comment (mekk1t) [reply DigitalOffice/Services/FileService/Startup.cs]: @DarkFelas answered in another PR:
static allows to skip "this." pointer and the method is easier to discover by the compiler (?)
can't say for sure, but that's the explanation I managed to find.

Are there any downsides using static methods?

Comment (MihailKulikov) [reply DigitalOffice/Services/FileService/Startup.cs]: Static modifier was removed.

**Review by DarkFelas — APPROVED** [review]

**Pull request merged**